### PR TITLE
Modernize python 2 codes

### PR DESCRIPTION
**Pull request by cclauss**
* Use __print()__ function in both Python 2 and Python 3
    * __print()__ is a function in Python 3.
* Old style exceptions --> new style for Python 3
    * Python 3 treats old style exceptions as syntax errors but new style exceptions work as expected in both Python 2 and Python 3.

Once this lands, it would be good to revendor it into OpenLibrary at
* https://github.com/internetarchive/openlibrary/tree/master/vendor

### CODE REVIEW

File: infogami/plugins/wikitemplates/forms.py [hunk 1]
```diff
@@ -17,4 +18,4 @@ def render(self):
 )
 
 if __name__ == "__main__":
-    print template_preferences().render()
\ No newline at end of file
+    print(template_preferences().render())
```
Comment (tfmorris) [inline infogami/plugins/wikitemplates/forms.py]: Add newline please

**Comment by cclauss** [comment]
The Python 3 syntax error is just like internetarchive/openlibrary#1468

File: infogami/plugins/wikitemplates/forms.py (hunk 1 above)
Comment (cclauss) [reply infogami/plugins/wikitemplates/forms.py]: Added.

**Comment by cclauss** [comment]
Added issues in upsream..  https://github.com/infogami/infogami/issues

**Comment by mekarpeles** [comment]
Thank you @EdwardBetts, @tfmorris, @hornc, and @cclauss. I resolved the merge conflicts and am merging these changes.

**Pull request merged**